RecoLocalTracker/SiPixelRecHits/src/PixelCPEClusterRepair.cc

```diff
@@ -336,8 +347,8 @@ PixelCPEClusterRepair::callTempReco2D( DetParam const & theDetParam,
    {
       LogDebug("PixelCPEClusterRepair::localPosition") <<
       "reconstruction failed with error " << theClusterParam.ierr << "\n";
-       theClusterParam.templProbY_ = theClusterParam.templProbX_ = theClusterParam.templProbQ_ = 1.0f;
-       theClusterParam.templQbin_ = 0;
+       theClusterParam.templProbY_ = theClusterParam.templProbX_ = theClusterParam.templProbQ_ = 0.;
+       theClusterParam.qBin_ = 0;
```

Comment:
I'm a bit lost with these "ClusterParam"...
Just to check: there remain still two ```theClusterParam.templQbin_``` in this class: could you please verify and confirm that they were not forgotten?

Reply:
Yes there is a theClusterParam.templQbin_ but as far as I can tell it is not used at all (?). All the calls to the low level reco code are made just using theClusterParam.qBin_ which is inherited from the base class. I can try to get rid of templQbin_ for clarity if you want.

Reply:
Please get rid of it if not needed, thank you